scriptmodules/ports/steamlink.sh

```diff
@@ -32,5 +32,22 @@ function remove_steamlink() {
 }
 
 function configure_steamlink() {
+    local sl_dir="$home/.local/share/SteamLink"
+    local slconfig_dir="$home/.local/share/Valve Software/Streaming Client"
+
+    mkUserDir "$sl_dir"
+    mkUserDir "$slconfig_dir"
```

Comment:
if home/.local/share/Valve Software doesn't exist it will be made with root permissions so a chown is needed. (as the mkUserDir does a mkdir -p as root, but only chowns the directory passed, not the parent directories.

Reply:
Should be sorted now.